source/uwp/Renderer/lib/ActionHelpers.cpp

```diff
@@ -677,21 +676,44 @@ namespace AdaptiveNamespace::ActionHelpers
 
         if (action != nullptr)
         {
-            // If we have an action, use the title for the AutomationProperties.Name
+            // If we have an action, use the title for the AutomationProperties.Name and tooltip
             HString title;
             THROW_IF_FAILED(action->get_Title(title.GetAddressOf()));
 
-            ComPtr<IDependencyObject> buttonAsDependencyObject;
-            THROW_IF_FAILED(button.As(&buttonAsDependencyObject));
+            if (title.IsValid())
+            {
```

Comment:
Where does the tooltip show up by default? Where/How do we configure that and is that controllable by the host?

Reply:
By default, a ToolTip is displayed centered above the pointer. ([https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tooltips#create-a-tooltip](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tooltips#create-a-tooltip)). There is some placement configuration available in XAML, but i don't think it's something that makes sense for the host to control, as presumably any positioning concerns would be card specific. I suppose we could expose positioning configuration to the card schema, but i'm not convinced that's necessary (and would be a schema change, which this currently isn't). 

---
In reply to: [523177305](https://github.com/microsoft/AdaptiveCards/pull/5044#discussion_r523177305) [](ancestors = 523177305)